README.md

```diff
@@ -191,23 +221,35 @@ purposes, you can create a self-signed localhost server certificate using
 OpenSSL:
 
 ```
+mkdir config
 openssl req -x509 -nodes -newkey ec \
     -pkeyopt ec_paramgen_curve:secp521r1 \
     -pkeyopt ec_param_enc:named_curve  \
     -subj '/CN=localhost' \
-    -keyout key.pem -out cert.pem -sha256 -days 3650 \
+    -keyout config/tls-key.pem -out config/tls-cert.pem -sha256 -days 3650 \
     -addext "extendedKeyUsage = serverAuth" \
     -addext "keyUsage = digitalSignature, keyCertSign, keyAgreement"
 ```
 
-This command creates an unencrypted private key, `key.pem`.
+This command creates an unencrypted private key, `config/tls-key.pem`.
 
 ### Running the proxy server
 
-You can start the proxy server using the following command:
+The proxy server can be run using the following command:
+
+```bash
+tesla-http-proxy -tls-key config/tls-key.pem -cert config/tls-cert.pem -key-file config/fleet-key.pem -port 4443
+```
+
+It can also be run using Docker:
 
 ```bash
-tesla-http-proxy -tls-key key.pem -cert cert.pem -port 4443
+# option 1: using docker run
+docker pull tesla/vehicle-command:latest
+docker run -v ./config:/config -p 4443:4443 tesla/vehicle-command:latest -tls-key /config/tls-key.pem -cert /config/tls-cert.pem -key-file /config/fleet-key.pem -host 0.0.0.0 -port 4443
```

Comment:
This exposes 4433 on the host, right? Let's make the example command use `-p 127.0.0.1:4433:4433` so people who are unfamiliar with docker don't inadvertently expose the proxy. We can add directions to omit the localhost IP if exposing the service is intentional.